index.js

```diff
@@ -2,113 +2,113 @@ const builder = require('botbuilder')
 let lib = new builder.Library('FormBuilder')
 
 lib.dialog('/', [
-  (session, args) => {
-    if (args !== null && typeof args !== 'undefined') {
-      if (!args.questions) {
-        return session.error('Missing questions array')
-      } else {
-        session.dialogData.questions = args.questions
-      }
-      session.dialogData.entities = args.entities ? args.entities : false
-      session.dialogData.index = args.index ? args.index : 0
-      session.dialogData.form = args.form ? args.form : {}
-      session.dialogData.reprompt = args.reprompt ? args.reprompt : false
-    } else {
-      return session.error('Arguments were not supplied or were undefined')
-    }
+    (session, args) => {
+        if (args !== null && typeof args !== 'undefined') {
+            if (!args.questions) {
+                return session.error('Missing questions array')
+            } else {
+                session.dialogData.questions = args.questions
+            }
+            session.dialogData.entities = args.entities ? args.entities : false
+            session.dialogData.index = args.index ? args.index : 0
+            session.dialogData.form = args.form ? args.form : {}
+            session.dialogData.reprompt = args.reprompt ? args.reprompt : false
+        } else {
+            return session.error('Arguments were not supplied or were undefined')
+        }
 
-    let index = session.dialogData.index
-    let questions = session.dialogData.questions
+        let index = session.dialogData.index
+        let questions = session.dialogData.questions
 
-    // Check if entities exists
-    if (session.dialogData.entities) {
-      // If the entities exists and it possesses the property for this question, send a confirm prompt
-      if (session.dialogData.entities.hasOwnProperty(questions[index].field)) {
-        var prompt = questions[session.dialogData.index].prompt
-        // Replace placeholder text with user data
-        prompt = prompt.replace('{' + questions[index].field + '}', session.dialogData.entities[questions[index].field])
-        builder.Prompts.confirm(session, prompt)
-      } else {
-        if (!session.dialogData.reprompt) {
-          builder.Prompts.text(session, questions[index].question)
+        // Check if entities exists
+        if (session.dialogData.entities) {
+            // If the entities exists and it possesses the property for this question, send a confirm prompt
+            if (session.dialogData.entities.hasOwnProperty(questions[index].field)) {
+                var prompt = questions[session.dialogData.index].prompt
+                // Replace placeholder text with user data
+                prompt = prompt.replace('{' + questions[index].field + '}', session.dialogData.entities[questions[index].field])
+                builder.Prompts.confirm(session, session.localizer.gettext(session.preferredLocale(), prompt))
```

Comment:
I'm not 100% sure about this. I'm not too familiar with the localiser functions, but from what I've read it goes and fetches a string from a separate JSON file. The prompt variable here is being built from the question object that is fed into the module. If you wanted to go multilingual, wouldn't you fetch the different language version on line 27 before you add the entity data into the string?

I should be able to take a look into this in more detail on Monday.